crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -88,12 +99,15 @@ use crate::core::ribosome::host_fn::count_links::count_links;
 use holochain_types::wasmer_types::WASM_METERING_LIMIT;
 use holochain_types::zome_types::GlobalZomeTypes;
 use holochain_types::zome_types::ZomeTypesError;
+use holochain_wasmer_host::module::InstanceWithStore;
```

Comment:
Why is this an instance with store now?

Reply:
@jost-s because that's how wasmer 3.x+ works, you need to track the store yourself